index.html

```diff
@@ -0,0 +1,491 @@
+            <li><a class="decoration-none underline" href="#">Automobiles</a></li>
+            <li><a class="decoration-none underline" href="#">Games</a></li>
+            <li><a class="decoration-none underline" href="#">Education</a></li>
+            <li><a class="decoration-none underline" href="#">Food</a></li>
+            <li><a class="decoration-none underline" href="#">Health</a></li>
+            <li><a class="decoration-none underline" href="#">Jobs</a></li>
+            <li><a class="decoration-none underline" href="#">Love</a></li>
+            <li><a class="decoration-none underline" href="#">Magazine</a></li>
+            <li><a class="decoration-none underline" href="#">Parenting</a></li>
+            <li><a class="decoration-none underline" href="#">Real Estate</a></li>
+            <li><a class="decoration-none underline" href="#">Recipes</a></li>
+            <li><a class="decoration-none underline" href="#">Style</a></li>
+            <li><a class="decoration-none underline" href="#">T Magazine</a></li>
+            <li><a class="decoration-none underline" href="#">Travel</a></li>
+          </ul>
+        </div>
+        <div class="more">
+          <h3>MORE</h3>
+          <ul>
+            <li><a class="decoration-none underline" href="#">Reader Center</a></li>
+            <li><a class="decoration-none underline" href="#">Wirecutter</a></li>
+            <li><a class="decoration-none underline" href="#">Live Events</a></li>
+            <li><a class="decoration-none underline" href="#">The Learning Network</a></li>
+            <li><a class="decoration-none underline" href="#">Tools & Services</a></li>
+            <li><a class="decoration-none underline" href="#">Multimedia</a></li>
+            <li><a class="decoration-none underline" href="#">Photography</a></li>
+            <li><a class="decoration-none underline" href="#">Video</a></li>
+            <li><a class="decoration-none underline" href="#">Newsletters</a></li>
+            <li><a class="decoration-none underline" href="#">TimesMachine</a></li>
+            <li><a class="decoration-none underline" href="#">NYT Store</a></li>
+            <li><a class="decoration-none underline" href="#">Times Journeys</a></li>
+            <li><a class="decoration-none underline" href="#">Manage My Account</a></li>
+          </ul>
+        </div>
+        <div class="subs">
+          <h3>SUBSCRIBE</h3>
+          <ul class="fsize">
+            <li class="underline"><i class="fas fa-scroll"></i><a class="decoration-none" href="#">Home Delivery</a></li>
+            <li class="underline"><i class="fas fa-lira-sign"></i><a class="decoration-none" href="#">Digital Subscriptions</a></li>
+            <li class="underline"><i class="fas fa-chess-board"></i><a class="decoration-none" href="#">Games</a></li>
+            <li class="underline"><i class="fas fa-utensils"></i><a class="decoration-none" href="#">Cooking</a></li>
```

Comment:
- [ ] [Optional] please adjust the icons to be the same size as the text, and make them inside the `a` tag